_posts/2018-05-30-nanopolish.md

```diff
@@ -0,0 +1,23 @@
+---
+site: freiburg
+tags: [tools]
+title: "Nanopolish tools for signal-level analysis of Oxford Nanopore sequencing data is now available in Galaxy"
+---
+
+![Nanopolish](https://github.com/jts/nanopolish) is a popular tool suite for working with raw and processed of Nanopore sequencing platform. We are pleased to announce that the demanding tools have been integrated into the Galaxy platform and are available on [our server](https://usegalaxy.eu).
```

Comment:
@mmiladi with this sentence is something wrong. `working with raw and processed READS`?

Reply:
Why not taking this sentence from github as well? `Nanopolish can calculate an improved consensus sequence for a draft genome assembly, detect base modifications, call SNPs and indels with respect to a reference genome and more`

Reply:
I thought might be better to not copy :) But right why not :) Please go ahead and replace it